doc/source/user_guide/launching_ansys_fluent.rst

```diff
@@ -3,16 +3,18 @@
 Launch or connect to Fluent
 ===========================
 You can use the :func:`launch_fluent() <ansys.fluent.core.launcher.launcher.launch_fluent>`
-method to start Fluent from Python in gRPC mode. This code starts Fluent in the background
+function to start Fluent from Python in gRPC mode. This code starts Fluent in the background
 so that commands can be sent to Fluent from the Python interpreter:
 
 .. code:: python
 
-    import ansys.fluent.core as pyfluent
-    solver = pyfluent.launch_fluent(mode="solver")
+  >>> import ansys.fluent.core as pyfluent
+  >>> solver = pyfluent.launch_fluent(
```

Comment:
I think for this line: **solver = pyfluent.launch_fluent()** makes more sense. Later we are explaining the modes.